app/helpers/queue_classic_admin/application_helper.rb

```diff
@@ -1,9 +1,9 @@
 module QueueClassicAdmin
   module ApplicationHelper
-    def has_qc_later?
-      QC.respond_to?(:enqueue_in) &&
-        defined?(QC::Later) &&
-        ActiveRecord::Base.connection.table_exists?(QC::Later::TABLE_NAME)
+
+    def has_qc_3_1?
```

Comment:
I would personally prefer a name reflecting what we're actually doing, which is looking for support of scheduling. Something like `has_scheduling?` would make more sense to me. Thoughts?